### PR TITLE
Add skyrim and dependencies

### PR DESCRIPTION
**Pull request by isuruf**
Checklist
- [ ] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [ ] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/5eddbd7fc9d1502169089da06c3688d9759be978/recipes/example/meta.yaml#L64-L73) for an example).
- [ ] Source is from official source.
- [ ] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [ ] If static libraries are linked in, the license of the static library is packaged.
- [ ] Package does not ship static libraries. If static libraries are needed, [follow CFEP-18](https://github.com/conda-forge/cfep/blob/main/cfep-18.md).
- [ ] Build number is 0.
- [ ] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [ ] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [ ] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by timkpaine** [comment]
agreed

**Comment by isuruf** [comment]
@beckermr, the linter fails here

**Comment by beckermr** [comment]
fixed! thanks for the bug report!

File: recipes/earth2mip/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+  entry_points:
+    - fcn = earth2mip.networks.fcn:load
+    - dlwp = earth2mip.networks.dlwp:load
+    - pangu = earth2mip.networks.pangu:load
+    - pangu_6 = earth2mip.networks.pangu:load_6
+    - pangu_24 = earth2mip.networks.pangu:load_24
+    - fcnv2_sm = earth2mip.networks.fcnv2_sm:load
+    - graphcast = earth2mip.networks.graphcast:load_time_loop
+    - graphcast_operational = earth2mip.networks.graphcast:load_time_loop_operational
+    - graphcast_small = earth2mip.networks.graphcast:load_time_loop_small
```
Comment (zklaus) [inline recipes/earth2mip/meta.yaml:27]: Note that these are not script entry points, but rather plugin-type (see [here](https://github.com/NVIDIA/earth2mip/blob/18482abdd7be3acaa0c9b5b400260bae582ed3a8/pyproject.toml#L90)). Do we still need to list them here?

**Review by zklaus — APPROVED** [review]
One small clarification request regarding entrypoints. Otherwise looks good to me.

File: recipes/earth2mip/meta.yaml (hunk 1 above)
Comment (isuruf) [reply recipes/earth2mip/meta.yaml:27]: Huh. I didn't notice it. This was created by grayskull and I didn't look thoroughly. Will fix.

**Review by zklaus — APPROVED** [review]

**Comment by ocefpaf** [comment]
skyrim

**Comment by isuruf** [comment]
Thanks

**Pull request merged**